movePlayerScript.js

```diff
@@ -0,0 +1,16 @@
+var storedKeyPress = 'a';
+
+
+// This function moves a html element across the screen. It can be have the keypress code and the html element entered as parameters. It then returns the storedKeyPress variable to prevent the same key being pressed twice.
```

Comment:
This documentation goes over the screen width. There is a line to tell you where the screen length in PHP storm. 
please change